### ISSUE TITLE
contmix

### ISSUE THREAD

**Issue by Century-lu** [LMMMEng/OverLoCK#12]
在你的代码中哪一个类是contmix块

**Comment by LMMMEng** [comment]
从这里开始：https://github.com/LMMMEng/OverLoCK/blob/f902faf3dd99341a68a6c29d77e91054bf44c05a/models/overlock.py#L505

**Comment by 11313131** [comment]
> 从这里开始：
> 
> [OverLoCK/models/overlock.py](https://github.com/LMMMEng/OverLoCK/blob/f902faf3dd99341a68a6c29d77e91054bf44c05a/models/overlock.py#L505)
> 
> Line 505 in [f902faf](/LMMMEng/OverLoCK/commit/f902faf3dd99341a68a6c29d77e91054bf44c05a)
> 
>  query, key = torch.split(x_f, split_size_or_sections=[C, C_h], dim=1)

那作者跟新的contmix.py中咋是从 x = self.dwconv1(x)这里开始的呢，这不跟class DynamicConvBlock(nn.Module):里开始的一样呢，麻烦作者回答这个疑问，谢谢！
